logger.js

```diff
@@ -51,8 +51,16 @@ module.exports = function(config) {
 
 	config.errorLevel = config.errorLevel || 'log';
 
-	child_process.execSync(`mkdir -p ${path.dirname(config.filename)}`);
-	var log_file = fs.createWriteStream(config.filename, { flags: 'a' });
+	const logsSubDirectory = config.network || '';
+	const logsDirectory = path.join(
+		path.dirname(config.filename),
+		logsSubDirectory
```

Comment:
Here it makes things confusing. The log file path is a node configuration option. User set it to some path, and you changed that path runtime, make the user wondering where his log file is. 

So right way to do is to just add a custom configuration override for the network with different log file paths.